src/bin/cmd/wallet_args.rs

```diff
@@ -621,18 +615,17 @@ pub fn parse_process_invoice_args(
 	// file input only
 	let tx_file = parse_required(args, "input")?;
 
-	// Now we need to prompt the user whether they want to do this,
-	// which requires reading the slate
-	#[cfg(not(test))]
-	let adapter = FileWalletCommAdapter::new();
-	#[cfg(not(test))]
-	let slate = match adapter.receive_tx_async(&tx_file) {
-		Ok(s) => s,
-		Err(e) => return Err(ParseError::ArgumentError(format!("{}", e))),
-	};
+	if cfg!(not(test)) {
```

Comment:
When using `#[cfg(not(test))]`, tests were compiling and running successfully, but non-test builds were failing. This change makes sure type checks take place in both test mode and non-test mode.